packages/forms-web-app/src/views/manage-appeals/request-new-code.njk

```diff
@@ -0,0 +1,56 @@
+{% extends "../layouts/householder-banner/main-no-back-link.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% set title="Request a new code - Manage Appeals planning decision - GOV.UK" %}
+
+{% block pageTitle %}
+  {% if errors %}
+    {% set title = "Error: " + title %}
+  {% endif %}
+  {{ title }}
+{% endblock %}
+
+{% block content %}
+  {% if errors %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errorSummary
+    }) }}
+  {% endif %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">Request a new code</h1>
+      <p class="govuk-body">Emails sometimes take a few minutes to arrive. If you do not receive the code, you can request a new one.</p>
+      <form action="" method="post" novalidate>
+         {% call govukFieldset({
+          legend: {
+            text: "Enter your email address",
+            classes: "govuk-fieldset__legend--l",
+            isPageHeading: true
+          }
+        }) %}
+
+        {{ govukInput({
```

Comment:
Should we be having an email on this field.
Think we should already have this available, unless this is the entry point.

In my head we were still waiting for a story for entering email, can see you left a message in teams about this so will double check my assumptions

Reply:
This was requested by Paul G and Chris Hy